docs/deploy-configure.md

```diff
@@ -24,12 +24,58 @@ To deploy your first replica set:
    ```
    kubectl get mongodbcommunity --namespace <my-namespace>
    ```
-4. Connect clients to the MongoDB replica set:
+
+4. The Community Kubernetes Operator creates secrets that contains user's connection strings and credentials.
```

Comment:
there can be multiple users so I believe this should be `that contain users' connection strings` (I could be mistaken!)

Reply:
@chatton I had it this way at first. Your comment affirms that I should go with that. updated.